subhub/log.py

```diff
@@ -1,62 +1,147 @@
-import logging.handlers
-from pythonjsonlogger import jsonlogger
+"""
+Usage example:
+    from logger import get_logger
+    log = get_logger()
+    log.info('my_event', my_key1='val 1', my_key2=5, my_key3=[1, 2, 3], my_key4={'a': 1, 'b': 2})
+List of metadata keys in each log message:
+    event
+    func
+    level
+    lineno
+    module
+    time_human_localized
+    time_human_utc
+Requirements: pytz
+Limitations: multithreading is supported but not multiprocessing.
+"""
+
+import os
+import sys
+import time
+import uuid
 import datetime
+import inspect
+import logging
+import logging.config
+import platform
+import tempfile
+import threading
+import collections
+import structlog
+import pytz
+
+from subhub.cfg import CFG
+
+
+IS_CONFIGURED = False
+EVENT_UUID = str(uuid.uuid4())
+LOGGING_CONFIG = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "handlers": {"console": {"()": logging.StreamHandler, "level": CFG.APP_LOG_LEVEL}},
+    "loggers": {
+        CFG.APP_PROJNAME: {
+            "propagate": False,
+            "handlers": ["console"],
+            "level": "DEBUG",
+        }
+    },
+}
+
+
+def _event_uppercase(logger, method_name, event_dict):
+    event_dict["event"] = event_dict["event"].upper()
+    return event_dict
+
+
+def _add_timestamp(logger, method_name, event_dict):
+    dt_utc = datetime.datetime.fromtimestamp(time.time(), datetime.timezone.utc)
+    event_dict["timestamp_utc"] = dt_utc.isoformat()
+    event_dict["timestamp_pacific"] = dt_utc.astimezone(
+        pytz.timezone("US/Pacific")
```

Comment:
I'm not sure how beneficial it is to have this locked to Pacific timezone.

Reply:
yeah the original code had it localized to US/Eastern; I just figured so many of us (not you though) are in the Pacific TZ. I can remove it though; thoughts?

Reply:
My vote is only show UTC, not a deal breaker though.